pkg/skuba/actions/node/bootstrap/bootstrap.go

```diff
@@ -48,7 +49,7 @@ func Bootstrap(bootstrapConfiguration deployments.BootstrapConfiguration, target
 	if clientSet, err := kubernetes.GetAdminClientSet(); err == nil {
 		_, err := clientSet.ServerVersion()
 		if err == nil {
-			fmt.Printf("[bootstrap] node %q has already the core components bootstrapped\n", target.Target)
+			klog.Errorf("node %q has already the core components bootstrapped\n", target.Target)
```

Comment:
Not sure this is really an error. 
Second question is it really related to the PR. [2]

Reply:
1. I will change to klog.Infof. 

2. You are correct. This change is not related to the title `Ssh errors should show errors regardless of v level`.
While I was testing the log output. I saw the inconsistency and I changed it.
The log output should be consistent with timestamp and file name in the way that klog was configured.
```
I1016 14:21:02.314978   19390 bootstrap.go:162] downloading secrets from bootstrapped node "10.86.0.191"
I1016 14:21:03.991743   19390 bootstrap.go:72] deploying core add-ons on node "10.86.0.191"
[bootstrap] applying init configuration to node
```
And this should be an error because this cannot do bootstrap even though it can cause from user's mistakes.
And I am not changing non-error to error here. I am adding one more step.